### PR TITLE
Update thief.md

### PR DESCRIPTION
**Pull request by jetfire158**

### CODE REVIEW

File: content/theme/characters/archetypes/outlaws/thief.md [hunk 1]
```diff
@@ -2,4 +2,41 @@
...
+  presence: 7
+  reflex: 9
+
+skills:
+  awareness:
+    conceal: 4
+    sneak: 4
+    hide: 4
+    disguise: 3
+    forge: 2
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/outlaws/thief.md:21]: seems like a big missed opportunity to use `pick lock`, and `steal`, considering those are the two bread-and-butter skills of any thief.

File: content/theme/characters/archetypes/outlaws/thief.md [hunk 2]
```diff
@@ -2,4 +2,41 @@
...
+  coordination:
+    pilot: 3
+    shoot: 3
+    drive: 3
+  knowledge:
+    streetwise: 3
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/outlaws/thief.md:29]: probably also want `security`; it hasn't been written up yet, but it basically pertains to electronic security systems.

**Review by jeffhuyghebaert — APPROVED** [review]
Add in those few skills mentioned and this should be good to go.

**Review by jetfire158 — COMMENTED** [review]
updated

**Pull request merged**